requests_cache/response.py

```diff
@@ -160,6 +173,23 @@ def stream(self, amt=None, **kwargs):
 AnyResponse = Union[Response, CachedResponse]
 
 
+def format_datetime(value: Optional[datetime]) -> str:
+    return value.strftime(DATETIME_SHORT_FORMAT) if value else "N/A"
+
+
+def format_file_size(n_bytes: int) -> str:
+    """Convert a file size in bytes into a human-readable format"""
+    filesize = float(abs(n_bytes))
```

Comment:
Why `abs`? I rather have `ValueError` be thrown.

Reply:
I was potentially going to reuse that function in some other places. I'll just remove the `abs` for now.